src/main/java/net/vivialconnect/util/ProjectProperties.java

```diff
@@ -0,0 +1,34 @@
+package net.vivialconnect.util;
+
+import java.util.Properties;
+import java.io.InputStream;
+import java.io.IOException;
+
+import org.springframework.core.io.ClassPathResource;
```

Comment:
Try not to include this extra library just for this.

Here's a link on how to load a resource from the classpath: https://www.mkyong.com/java/java-read-a-file-from-resources-folder/

This will allow you to remove the dependency on spring, which could turn out huge.

Reply:
Okay, I made the switch. Thanks for the info.